sormas-backend/src/main/java/de/symeda/sormas/backend/util/PatchHelper.java

```diff
@@ -0,0 +1,330 @@
+	 * @throws JsonProcessingException
+	 * @throws IllegalAccessException
+	 */
+	private static <T extends HasUuid> void updateObjectList(T existingObject, JsonNode jsonObjectFieldNode, Field existingObjectField) {
+		Class listElementClass = getParameterizedType(existingObjectField);
+		List tempNewElementList = new ArrayList();
+
+		for (JsonNode listElement : jsonObjectFieldNode) {
+			T updatedObjectList = addOrReplaceListElement(existingObject, existingObjectField, listElementClass, listElement);
+			if (updatedObjectList != null) {
+				tempNewElementList.add(updatedObjectList);
+			}
+		}
+
+		if (existingObjectField.getType().isAssignableFrom(List.class)) {
+			Object existingObjectFieldInstance = null;
+			try {
+				existingObjectFieldInstance = existingObjectField.get(existingObject);
+			} catch (IllegalAccessException e) {
+				throw new RuntimeException(
+					"Cannot access field " + existingObjectField.getName() + " on the object of type: " + existingObject.getClass().getName());
+			}
+			if (existingObjectFieldInstance != null) {
+				Collection existingElementList = (Collection) existingObjectFieldInstance;
+				existingElementList.clear();
+				existingElementList.addAll(tempNewElementList);
+			} else {
+				try {
+					existingObjectField.set(existingObject, tempNewElementList);
+				} catch (IllegalAccessException e) {
+					e.printStackTrace();
+					throw new RuntimeException(
+						"Cannot access field " + existingObjectField.getName() + " on the object of type: " + existingObject.getClass().getName());
+				}
+			}
+		} else if (existingObjectField.getType().isArray()) {
+			try {
+				existingObjectField.set(existingObject, tempNewElementList.toArray());
+			} catch (IllegalAccessException e) {
+				e.printStackTrace();
+				throw new RuntimeException(
+					"Cannot access field " + existingObjectField.getName() + " on the object of type: " + existingObject.getClass().getName());
+			}
+		} else {
+			throw new RuntimeException(
+				"The list of objects expect to have a type that inherits from Collection or to be an array! The current list object field "
+					+ existingObjectField.getName() + " has a type of: " + existingObjectField.getType().getName());
+		}
+	}
+
+	/**
+	 * This method update an existing object field. If the existingObject field is NULL a new instance is created.
+	 * If the existing object field is not NULL the postUpdate method is called on the object field
+	 * 
+	 * @param existingObject
+	 * @param jsonObjectFieldNode
+	 * @param existingObjectField
+	 * @param <T>
+	 * @throws IllegalAccessException
+	 * @throws JsonProcessingException
+	 */
+	private static <T extends HasUuid> void updateExistingObject(T existingObject, JsonNode jsonObjectFieldNode, Field existingObjectField) {
+		T existingFieldValue = null;
+		try {
+			existingFieldValue = (T) existingObjectField.get(existingObject);
+		} catch (IllegalAccessException e) {
+			e.printStackTrace();
+			throw new RuntimeException(
```

Comment:
please use `RuntimeException(String message, Throwable cause)` instead of `e.printStackTrace(); throw new RuntimeException`